eslint.config.js

```diff
@@ -0,0 +1,20 @@
+const { FlatCompat } = require('@eslint/eslintrc');
+
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: require('./node_modules/@balena/lint/config/eslint.config.js'),
+});
+
+module.exports = compat.config({
+	parserOptions: {
+		project: 'tsconfig.dev.json',
+		sourceType: 'module',
+	},
+	env: {
+		jest: true,
+	},
+	root: true,
+	rules: {
+		'no-restricted-imports': ['error', 'date-fns', 'lodash'],
+	},
+});
```

Comment:
as far as I know Page is pretty against custom config for a specific repo, did you discuss it with him ?

Reply:
I only focused on bumping balena-lint in this PR, so this rule was there before, but tbh blocking top level imports in the SDK & the UI is definitely something that we need to keep the bundle size small, while on the other hand it would legitimately be adding friction to all other projects...